Explorer/Assets/DCL/CharacterMotion/Components/CharacterRigidTransform.cs

```diff
@@ -14,29 +14,53 @@ namespace DCL.CharacterMotion.Components
     /// </summary>
     public class CharacterRigidTransform
     {
+        public struct MovementVelocity
```

Comment:
Can you check settings of your auto clean-up on commit? Mine changes the layout and removes redundant namespaces